src/Channel.cc

```diff
@@ -49,6 +49,7 @@ Channel::~Channel() {
 
 std::shared_ptr<Object> Channel::read() {
     AstObject *rawRet = reinterpret_cast<AstObject *>(astRead(getRawPtr()));
+    assertOK(rawRet);
```

Comment:
I don't think it takes an argument.

Reply:
It takes two optional arguments: https://github.com/lsst/astshim/blob/master/include/astshim/base.h#L108

Reply:
Signature says it takes up to two objects that should be deallocated if the error check fails.  We should really be capturing pointers like this in a smart pointer *immediately* and relying on destructors to take care of them, but that definitely seems out of scope for this ticket, and it's possible there was a valid reason for not doing so here.

Reply:
Weird, the others in this file don't seem to take any, but ok 😄